xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1416,10 +1346,10 @@
         <Parameter Name="value" Type="System.Text.RegularExpressions.Group" RefType="out" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">A string with the group name key to look for.</param>
+        <param name="value">The out parameter where the retrieved group will be saved if found; <see langword="null" /> if not found.</param>
```

Comment:
```suggestion
        <param name="value">When the method returns, the group whose name is <paramref name="key" />, if it is found; otherwise, <see langword="null" /> if not found.</param>
```